src/scripts/main.js

```diff
@@ -1,7 +1,33 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const par = document.body;
+
+  const divElem = document.createElement('div');
+
+  divElem.className = `notification ${type}`;
+  par.appendChild(divElem);
+
+  divElem.setAttribute('style',
+    `top: ${
+      posTop
+    }px; right: ${
+      posRight
+    }px; box-sizing: content-box;`);
```

Comment:
You can modify style properties directly, as follow:
`divElem.style.top = ${posTop}px;`